lib/transaction/transaction.go

```diff
@@ -84,6 +84,7 @@ func NewTransaction(source string, sequenceID uint64, ops ...Operation) (tx Tran
 }
 
 var TransactionWellFormedCheckerFuncs = []common.CheckerFunc{
+	CheckTransactionMaxOperations,
```

Comment:
It seems like confirming that it is full.
How about this `CheckTransactionOverOperationLimit`?